department-of-veterans-affairs/caseflow
# Add behavior for Bva Intake queue "Pending" tab

Resolves [CASEFLOW-2277](https://vajira.max.gov/browse/CASEFLOW-2277)

### Description
These changes allow the Bva org queue "Pending" tab to show all appeals in the predocket workflow

### Acceptance Criteria
- [ ] "Pending" tab in Bva Intake org queue shows all appeals in the pre docket flow
- [ ] This includes appeals assigned to CAMO, Program Offices and VISNs
- [ ] The default sorting of the appeals should be from oldest (total time) to newest
- [ ] The tab should update the status of the appeal (task owner) based on where the appeal is within the VHA workflow

### Testing Plan
1. Intake a predocket appeal (default behavior is that the appeal is now assigned to CAMO)
2. Repeat step 1 and assign this appeal out to a Program Office
3. '' except this time assign the appeal to a VISN
4. Ensure all 3 appeals show in the "Pending" tab
5. Forward the appeal with CAMO to Bva for docketing
6. Ensure that appeal no longer shows in the "Pending" tab and now shows in "Ready for Review"
